www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -806,19 +800,32 @@ function (declare, domClass, dom, style, fx, Toggler, keys, on, query, Memory,
     _formatReportDetails : function (report, reportDetails) {
       var res = [];
 
+      var lastBugEvent = null;
```

Comment:
This can be written like so:
```js
var lastBugEvent = reportDetails.pathEvents.length
  ? reportDetails.pathEvents[reportDetails.pathEvents.length - 1]
  : null;
```